.github/workflows/test.yml

```diff
@@ -24,6 +24,7 @@ jobs:
           - 0.6
```

Comment:
Should we remove version `0.4` and `0.5` (and perhaps also `0.6`) from our ever-growing test matrix?

Reply:
@ibnesayeed Yes. Would you be ok with keeping 0.7 and above?